apps/hashdotdev/src/pages/roadmap/technology-tree-data.tsx

```diff
@@ -184,12 +171,7 @@ export const technologyTreeData: TechnologyTreeNodeData[] = [
     body: "Freeform drag-and-drop canvases for blocks",
     parentIds: ["15"],
     status: "in-progress",
```

Comment:
possibly? it's there, it works, it needs more work
```suggestion
    status: "working-poc",
```